bin/bundle-extensions

```diff
@@ -1,39 +1,43 @@
 #!/usr/bin/env python
-
+"""Copy bundle extension files to the client/app/extension directory"""
 import os
-from subprocess import call
-from distutils.dir_util import copy_tree
-
-from pkg_resources import iter_entry_points, resource_filename, resource_isdir
-
+from pathlib2 import Path
+from shutil import copy
 
+from redash import create_app, extensions
```

Comment:
Importing from `redash` currently breaks when no Redis server is available. We should probably fix this regardless, but just note that this code won't work in a Docker build execution, which is where it was previously used.

Reply:
Hmm, yeah, that'll be a problem. I think it would make sense to somehow make sure the importing the redash package doesn't have import time side-effects.

Reply:
I agree, I just didn't want this to be a dependency of your work here.

Reply:
No problem, I think this is something that we need to do anyway, so there is #3601 :)